### ISSUE TITLE
为: 中国不是欧洲的救世主 一文而来

### ISSUE THREAD

**Issue by hooper-hc** [neruthes/TheWestTimes-Forum#1]
我觉得不然, 中国抗疫并没有因为国别而有政治操作. 中国的抗疫援助是非常科学的.

首先中国抗疫模式和体制相关性极大, 所以想要中国帮助抗疫的国家在意识形态上, 必须有所共鸣, 才有可能接受中国的抗疫模式. 意识形态上不认同,有误解甚至故意丑化, 中国贸然去支援, 怎必然结果就是浪费宝贵的医疗资源和抗疫经验.  因为中国模式根本落实不下去. 

同时英国美国作为长期在意识形态上丑化中国的老牌国家, 民众在接受中国抗疫模式上, 必然反弹巨大. 十分不利与英美的国家稳定繁荣. 如果强制输送中国模式去英国, 必然会落实错误的意识形态, 导致英国的意识形态崩塌, 国家分裂, 会有很多地区开始公投........ 
我们权衡之下, 还是不要支援英国. 这对英国不好.

其次我也说了, 英美历来是强国, 医疗体系远比中国健全. 站在全人类的高度来讲, 英美完全有能力抗疫, 这疫情在英国根本不算什么, 我们应该帮助没有能力抗疫的国家. 人类才能早日战胜疫情. 英国都是绅士, 应该懂得怜悯没有能力抗疫的人群. 

---

最后如果英国将来有一天疫情大爆发了, 请先做一件事:  告诉你的民众, 中国抗疫模式是成功的, 是科学的. 让民众能够接受, 然后在引入中国帮助抗疫, 这才是更加科学.

**Comment by hooper-hc** [comment]
I don't think so. China's anti-epidemic has no political operation because of its country. China's anti-epidemic assistance is very scientific.

First of all, China ’s anti-epidemic model and system are extremely relevant, so countries that want China to help fight the epidemic must have some ideological resonance before they can accept China ’s model of anti-epidemic. Ideologically disagree, misunderstood or even Intentional ugliness, China rushed to support, how the inevitable result is a waste of valuable medical resources and anti-epidemic experience. Because the Chinese model cannot be implemented at all.

At the same time, the United Kingdom and the United States have long been ideologically vilifying China's established countries. The public's acceptance of the Chinese anti-epidemic model will inevitably rebound greatly. It is very unfavorable to the stability and prosperity of the British and American countries. Ideology, leading to the collapse of Britain's ideology, the split of the country, there will be many regions to start the referendum ...
We weighed it and still don't support Britain. This is not good for Britain.

Secondly, I also said that Britain and the United States have always been strong countries, and the medical system is far more robust than China. From the perspective of all human beings, the United Kingdom and the United States are fully capable of fighting the epidemic. This epidemic is nothing in the United Kingdom. We should help those who are unable to fight the epidemic. Country. Humans can overcome the epidemic as soon as possible. Britain is a gentleman and should understand the pity of those who are unable to fight the epidemic.

Finally, if there is a major outbreak in the United Kingdom in the future, please do one thing first: Tell your people that China ’s epidemic prevention model is successful and scientific. Let the people accept it, and then introduce China to help fight the epidemic. Is more scientific.

**Comment by neruthes** [comment]
我希望读者们能够通过对比《多党轮流执政照样腐败》、《一党长期执政照样低效》这两个标题看到，我在西方时报系列内的文章，重点在于鼓吹西方特色资本主义的道路自信、制度自信。要达成的幽默效果是，东方与西方的攻守易势；过去是中国媒体抵抗西方价值观输入，我写的是西方媒体抵抗东方价值观输入。

抵抗东方价值观主要靠 3 个论点：

- 制度不符合国情
- 人口素质还不够高
- 防范新殖民主义

最新的《中国不是欧洲的救世主》，视角是西方官媒打击西方公知、鼓吹中国威胁论。这与反河殇运动、反公知运动是同构的。鼓吹中国有特殊利益诉求，与中国官媒鼓吹美国有特殊利益诉求是一致的。同样的论证角度、论证方法，由西方媒体来实践，对中国读者展现出了西方媒体一贯的商标和中国官媒一贯的叼飞盘。

至于中国是否有特殊利益诉求、是否想要输出价值观，其实是无关紧要的。并且我正是通过写这篇文章来论证这个问题如何无关紧要。
